### ISSUE TITLE
[Bug, Unity] mono[class] causes livesplit to hang

### ISSUE THREAD

**Issue by mitchell-merry** [just-ero/asl-help#1]
Full log: https://pastebin.com/42fgm6h0

Game is Night at the Gates of Hell. Unity 2021.3.2fl, and mono v3 (according to `asl-help`). A script as simple as the following is hanging (i.e. not responding, freezing, and then closing) LiveSplit on `mono["GameManager"]` (same behavior with `GetClass`).

```cs
state("Night at the Gates of Hell") { }

startup
{
	Assembly.Load(File.ReadAllBytes("Components/asl-help")).CreateInstance("Unity");
}

init
{
	vars.Helper.TryLoad = (Func<dynamic, bool>)(mono =>
	{
		var gm = mono["GameManager"];
		
		return true;
	});
}
```

**Comment by mitchell-merry** [comment]
I've also experienced this on [Late Work](https://gubebra.itch.io/late-work):
```cs
state("Late Work") { }

startup
{
	Assembly.Load(File.ReadAllBytes("Components/asl-help")).CreateInstance("Unity");
}

init
{
	vars.Helper.TryLoad = (Func<dynamic, bool>)(mono =>
	{
		var gm = mono["GameManagerScript"];

		return true;
	});
}
```

**Comment by just-ero** [comment]
Fixed in #6.
